### PR TITLE
feat: OpenAPI max file size

### PR DESCRIPTION
**Pull request by kgilpin**
## Demo

https://asciinema.org/a/v69jrmpHLOasAH27JagH1wBeR

### CODE REVIEW

File: packages/cli/src/cmds/openapi.ts [hunk 1]
```diff
@@ -111,6 +145,10 @@ export default {
     args.option('appmap-dir', {
       describe: 'directory to recursively inspect for AppMaps',
     });
+    args.option('max-size', {
+      describe: 'maximum AppMap size that will be processed, in filesystem-reported MB',
```
Comment (apotterri) [inline packages/cli/src/cmds/openapi.ts:149]: "filesystem-reported" seems like a slightly odd way to describe this: "filesystem-reported" as opposed to what? Maybe this should just be "in MB"?

**Review by apotterri — APPROVED** [review]
One minor wording change to consider, otherwise good-to-go.

**Comment by kgilpin** [comment]
Yeah I would be inclined to change it if builds didn’t take an hour :-)

**Comment by kgilpin** [comment]
It means that a MB is 1024*1024, not 1e6

**Pull request merged**